### PR TITLE
Add "My Reservations" page

### PR DESCRIPTION
**Pull request by kdeloach**
This splits the existing reservations page into 2 pages. Clicking
"Reservations" in the sidebar will display your currently reserved blockfaces.
Clicking the "New" button will take you to the next page to reserve
blockfaces.

### CODE REVIEW

**Comment by maurizi** [comment]
Looks good to me

**Comment by jwalgran** [comment]
+1 Layer works perfectly.

If the user has blockfaces reserved, it makes sense to set the map view to the extent of those blockfaces.     It should be relatively easy to get a queryset of the blockfaces and drop the extent into a view context. https://docs.djangoproject.com/en/1.7/ref/contrib/gis/geoquerysets/#extent

We can split this feature off into a separate issue if you like, or roll it into this PR.

**Comment by RickMohr** [comment]
Note that @maurizi just did this for the event list page, purely client side -- https://github.com/maurizi/nyc-trees/commit/08126836a6bfca3fbc03b1b359d3bc15be5b48e0

File: src/tiler/sql/user_reservations.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+(SELECT
```
Comment (steventlamb) [inline src/tiler/sql/user_reservations.sql]: for my education, how does this file get used? I can't find a reference to `user_reservations` in this diff.

**Comment by steventlamb** [comment]
+1

**Comment by maurizi** [comment]
@RickMohr Leaflet can't compute boundaries of tile layers client side.
We'll have to go with the approach Justin outlined of computing a bounding
box with Django.

File: src/tiler/sql/user_reservations.sql (hunk 1 above)
Comment (kdeloach) [inline src/tiler/sql/user_reservations.sql]: Map tile requests made by an authenticated user are prefixed with 'user_' (See https://github.com/kdeloach/nyc-trees/blob/topic/my-reservations/src/tiler/server.js#L99)

**Comment by kdeloach** [comment]
Cool, it looks like I can reuse a good amount of Mike's code for fitting the bounding box to the reservations extent, so I would like to do this after both PRs are merged.

**Pull request merged**